smartsim/_core/control/controller.py

```diff
@@ -497,7 +565,9 @@ def _launch_step(
             self._jobs.add_job(job_step.name, job_id, entity, is_task)
 
     def _create_batch_job_step(
-        self, entity_list: t.Union[Orchestrator, Ensemble, _AnonymousBatchJob]
+        self,
+        entity_list: t.Union[Orchestrator, Ensemble, _AnonymousBatchJob],
+        telemetry_dir: pathlib.Path,
```

Comment:
Same "don't love, but good enough for now" concern with param diving the telemetry_dir: @MattToast and/or @ankona remember to ticket this as well!!

Reply:
+1